### PR TITLE
RenderStatePackager: Fixed import feature

### PR DESCRIPTION
**Pull request by MikhailGorobets**

### CODE REVIEW

File: RenderStateNotation/src/RenderStateNotationParserImpl.cpp [hunk 1]
```diff
@@ -260,17 +260,23 @@ Bool RenderStateNotationParserImpl::ParseFile(const Char* FileName, IShaderSourc
 
     try
     {
-        RefCntAutoPtr<IFileStream> pFileStream;
-        pStreamFactory->CreateInputStream(FileName, &pFileStream);
+        if (m_Includes.find(FileName) == m_Includes.end())
```
Comment (TheMostDiligent) [inline RenderStateNotation/src/RenderStateNotationParserImpl.cpp:263]: You can use `if(m_Includes.find(FileName).second)` to save one `insert`

File: RenderStateNotation/src/RenderStateNotationParserImpl.cpp [hunk 2]
```diff
@@ -260,17 +260,23 @@ Bool RenderStateNotationParserImpl::ParseFile(const Char* FileName, IShaderSourc
...
+                LOG_ERROR_AND_THROW("Failed to open file: '", FileName, "'.");
 
-        if (!pFileStream)
-            LOG_ERROR_AND_THROW("Failed to open file: '", FileName, "'.");
+            auto pFileData = DataBlobImpl::Create();
+            pFileStream->ReadBlob(pFileData);
 
-        auto pFileData = DataBlobImpl::Create();
-        pFileStream->ReadBlob(pFileData);
+            if (!ParseString(static_cast<const char*>(pFileData->GetConstDataPtr()), static_cast<Uint32>(pFileData->GetSize()), pStreamFactory))
```
Comment (TheMostDiligent) [inline RenderStateNotation/src/RenderStateNotationParserImpl.cpp:274]: `StaticCast`

File: RenderStateNotation/src/RenderStateNotationParserImpl.cpp (hunk 1 above)
Comment (MikhailGorobets) [reply RenderStateNotation/src/RenderStateNotationParserImpl.cpp:263]: This isn't a map

Comment (TheMostDiligent) [reply RenderStateNotation/src/RenderStateNotationParserImpl.cpp:263]: [std::unordered_set::insert](https://en.cppreference.com/w/cpp/container/unordered_set/insert) works similar to std::unordered_map::insert

**Pull request merged**